eng/common/core-templates/steps/publish-logs.yml

```diff
@@ -56,3 +56,4 @@ steps:
       artifactName: PostBuildLogs
       continueOnError: true
       condition: always()
+      retryCountOnTaskFailure: 10 # for any logs being locked
```

Comment:
`eng\common` comes from dotnet/arcade. We shouldn't be modifying these files directly - instead they should go to dotnet/arcade and flown to all repos.

Reply:
right, thanks -- I have another PR there, this one was to make the same change automatically.
https://github.com/dotnet/arcade/pull/15593

I'll close this in favor of that